californium-core/src/main/java/org/eclipse/californium/core/network/EndpointManager.java

```diff
@@ -169,18 +168,17 @@ public synchronized void setDefaultEndpoint(Endpoint newEndpoint) {
 	 */
 	private Endpoint createDefaultEndpoint(String uriScheme) {
 		Endpoint endpoint;
-		NetworkConfig config = NetworkConfig.getStandard();
-
 		if (CoAP.COAP_SECURE_URI_SCHEME.equalsIgnoreCase(uriScheme)) {
-			LOGGER.config("Secure endpoint must be injected via setDefaultEndpoint(Scheme, Endpoint) to use the proper credentials");
+			LOGGER.config(
+					"Secure endpoint must be injected via setDefaultEndpoint(Scheme, Endpoint) to use the proper credentials");
 			return null;
 		} else if (CoAP.COAP_TCP_URI_SCHEME.equalsIgnoreCase(uriScheme)) {
-			TcpClientConnector connector = new TcpClientConnector(config.getInt(NetworkConfig.Keys.TCP_WORKER_THREADS),
-					config.getInt(NetworkConfig.Keys.TCP_CONNECT_TIMEOUT),
-					config.getInt(NetworkConfig.Keys.TCP_CONNECTION_IDLE_TIMEOUT));
-			endpoint = new CoapEndpoint(connector, config);
+			LOGGER.config(
+					"Tcp endpoint must be injected via setDefaultEndpoint(Scheme, Endpoint)");
+			return null;
 		} else if (CoAP.COAP_SECURE_TCP_URI_SCHEME.equalsIgnoreCase(uriScheme)) {
-			LOGGER.config("Secure tcp endpoint must be injected via setDefaultEndpoint(Scheme, Endpoint) to use the proper credentials");
+			LOGGER.config(
+					"Secure tcp endpoint must be injected via setDefaultEndpoint(Scheme, Endpoint) to use the proper credentials");
```

Comment:
If possible, could we correct the logging messages to `setDefaultEndpoint(Endpoint)` because the Scheme is read from the endpoint (and not longer passed in)? I forgot/overseen that, when I was changing the EndpointManager.